### DISCUSSION TITLE
NERF without tcnn?

### DISCUSSION THREAD

**Discussion by sinAshish** [nerfstudio-project/nerfstudio#2290] [Q&A]
Hello, I am  trying to train a nerf/sdf, but since the GPU is old, I'm unable to install TCNN. 
Therefore Im unable to use kplanes/neuralangelo, etc to create a nerf/sdf.

So, is there a way to use nerfstudio/sdfstudio without relying on tcnn? I'm not super concerned about the speed.

**Comment by tancik** [comment]
Yep, it will fallback to pytorch if TCNN isn't found. Make sure you are using the latest version.

**Comment by sinAshish** [comment]
oh, i guess this is only true for nerfstudio and not sdfstudio.
it seems like the pip release needs to be updated to reflect the changes proposed in a previous PR. where it fails to fall backs to using pytorch because of wrong position of `self.build_nn_modules()` in L104 of MLP.py
